functional-tests/step_definitions/custom_test_steps.js

```diff
@@ -0,0 +1,27 @@
+const webMethods = require('kakunin')
+
+const { When, Then } = require('kakunin');
```

Comment:
You can import it as `const { When, Then, methods } = require('kakunin');` and remove the first line from the file